scripts/prep_forecast.sh

```diff
@@ -94,12 +92,13 @@ DumpFields=${NEMSDumpFields:-false}   #Dump diagnostic netcdf fields from compon
 CPL_SLOW=${CPL_SLOW:-$DT_THERM_MOM6}  #slow coupling time step
 CPL_FAST=${CPL_FAST:-$DT_ATMOS}       #fast coupling time step 
 
-#Calculated Varaibles: 
+#Calculated Variables: 
 SYEAR=$(echo  $CDATE | cut -c1-4)
 SMONTH=$(echo $CDATE | cut -c5-6)
 SDAY=$(echo   $CDATE | cut -c7-8)
 SHOUR=$(echo  $CDATE | cut -c9-10)
 NTASKS_TOT=${NTASKS_TOT:-"$(( $ATMPETS+$OCNPETS+$ICEPETS ))"} #240
+NEARESTCDATE=$(echo $CDATE | cut -c1-8)00
```

Comment:
Probably stupid question, but why do we need this?

Reply:
The forecast case that @JessicaMeixner-NOAA implemented expects a start at 00z, our DA starts at 12z ... We'll clean that up one day, but for the time being, the first cycle for the DA uses a restart that is 12 hours off. Shocking, I know!